ui/src/kapacitor/components/RuleGraph.js

```diff
@@ -88,6 +102,17 @@ export const RuleGraph = React.createClass({
           break
         }
 
+        case 'not equal to': {
+          const width =
+            theOnePercent * (dygraph.yAxisRange()[1] - dygraph.yAxisRange()[0])
+          highlightStart = +rule.values.value - width
```

Comment:
doesn't seem to work when the values are strings.  It's pretty common to have tickscripts use 'not equal to' or 'equal to' for strings.

Reply:
Here is a specific example of usage: https://github.com/influxdata/chronograf/issues/865